.github/workflows/tests.yml

```diff
@@ -84,8 +84,9 @@ jobs:
         run: |
             set -ex
             source tools/github/before_install.sh
+            set -ex
```

Comment:
do we need "set -ex" a second time?

Reply:
becuase the source'd script removes it.... it was tricky to debug should i add a comment?

Reply:
Okay, that is what I suspected. Let's go ahead and get this in as is